.ci/test-r-package.sh

```diff
@@ -169,18 +169,19 @@ elif [[ $R_BUILD_TYPE == "cran" ]]; then
     if [[ "${TASK}" == "r-rchk" ]]; then
         echo "Checking R-package with rchk"
         mkdir -p packages
-        cp ${PKG_TARBALL} packages
+        cp "${PKG_TARBALL}" packages
         RCHK_LOG_FILE="rchk-logs.txt"
         docker run \
-            -v $(pwd)/packages:/rchk/packages \
+            -v "$(pwd)/packages:/rchk/packages" \
             kalibera/rchk:latest \
             "/rchk/packages/${PKG_TARBALL}" \
-        2>&1 > ${RCHK_LOG_FILE} \
+        > ${RCHK_LOG_FILE} 2>&1  \
         || (cat ${RCHK_LOG_FILE} && exit 1)
         cat ${RCHK_LOG_FILE}
 
         # the exceptions below are from R itself and not LightGBM:
         # https://github.com/kalibera/rchk/issues/22#issuecomment-656036156
+        # shellcheck disable=SC2046
         exit $(
             cat ${RCHK_LOG_FILE} \
```

Comment:
Will this fix the warning? 🤔 
```suggestion
        exit $(
            cat "${RCHK_LOG_FILE}" \
```